### PR TITLE
added explicit dependencies to jdtCompilerApt.

### PR DESCRIPTION
**Pull request by cdietrich**
added explicit dependencies to jdtCompilerApt.
fixes https://github.com/eclipse/xtext-maven/issues/9

Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
@spoenemann @svenefftinge  please review

File: org.eclipse.xtext.purexbase/xtend-gen/org/eclipse/xtext/purexbase/jvmmodel/PureXbaseJvmModelInferrer.java [hunk 1]
```diff
@@ -56,7 +56,8 @@ public String name(final Resource res) {
     return s.substring(0, _minus);
   }
   
-  public void infer(final EObject m, final IJvmDeclaredTypeAcceptor acceptor, final boolean prelinkingPhase) {
+  @Override
+public void infer(final EObject m, final IJvmDeclaredTypeAcceptor acceptor, final boolean prelinkingPhase) {
```
Comment (svenefftinge) [inline org.eclipse.xtext.purexbase/xtend-gen/org/eclipse/xtext/purexbase/jvmmodel/PureXbaseJvmModelInferrer.java]: was this changed accidentally?

Comment (cdietrich) [inline org.eclipse.xtext.purexbase/xtend-gen/org/eclipse/xtext/purexbase/jvmmodel/PureXbaseJvmModelInferrer.java]: yes ill try to get rid of it (maybe i need to disable xtend compiler)

Comment (cdietrich) [inline org.eclipse.xtext.purexbase/xtend-gen/org/eclipse/xtext/purexbase/jvmmodel/PureXbaseJvmModelInferrer.java]: its the damn save actions

File: org.eclipse.xtext.builder.standalone.tests/build.gradle [hunk 2]
```diff
@@ -3,6 +3,7 @@ dependencies {
 	compile project(':org.eclipse.xtext.ecore')
 	compile "org.eclipse.xtext:org.eclipse.xtext.testing:$versions.xtext"
 	compile "org.eclipse.jdt:org.eclipse.jdt.core:$versions.jdtCore"
+	compile "org.eclipse.jdt:org.eclipse.jdt.compiler.apt:$versions.jdtCompilerApt"
```
Comment (spoenemann) [inline org.eclipse.xtext.builder.standalone.tests/build.gradle]: It's not necessary to add the dependency here: compile dependencies in Gradle are always transitive.

Comment (cdietrich) [inline org.eclipse.xtext.builder.standalone.tests/build.gradle]: Why is jdt here

Comment (spoenemann) [inline org.eclipse.xtext.builder.standalone.tests/build.gradle]: Could probably be removed as well.

Comment (cdietrich) [inline org.eclipse.xtext.builder.standalone.tests/build.gradle]: I can do it but there mit be a reason => 2.11.1 ?

Comment (spoenemann) [inline org.eclipse.xtext.builder.standalone.tests/build.gradle]: We should be careful when changing the dependencies of published projects, but here we're talking about a test project. But never mind, the additional dependencies don't really hurt.

**Pull request merged**